Formula/restview.rb

```diff
@@ -0,0 +1,64 @@
+class Restview < Formula
+  include Language::Python::Virtualenv
+
+  desc "Viewer for ReStructuredText documents that renders them on the fly"
+  homepage "https://github.com/mgedmin/restview"
+  url "https://github.com/mgedmin/restview/archive/2.7.0.tar.gz"
+  sha256 "ab288234da5c6b3f7cd02d6112d0a020822f43423c086089efeeda2ffac5c388"
+
+  head "https://github.com/mgedmin/restview.git", :shallow => false
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  resource "bleach" do
+    url "https://files.pythonhosted.org/packages/1e/67/2562affb99e194cb4b0c0b88e661650d065fcf452d1108116a9530ed9cad/bleach-2.0.0.tar.gz"
+    sha256 "b9522130003e4caedf4f00a39c120a906dcd4242329c1c8f621f5370203cbc30"
+  end
+
+  resource "docutils" do
+    url "https://files.pythonhosted.org/packages/84/f4/5771e41fdf52aabebbadecc9381d11dea0fa34e4759b4071244fa094804c/docutils-0.14.tar.gz"
+    sha256 "51e64ef2ebfb29cae1faa133b3710143496eca21c530f3f71424d77687764274"
+  end
+
+  resource "html5lib" do
+    url "https://files.pythonhosted.org/packages/17/ee/99e69cdcefc354e0c18ff2cc60aeeb5bfcc2e33f051bf0cc5526d790c445/html5lib-0.999999999.tar.gz"
+    sha256 "ee747c0ffd3028d2722061936b5c65ee4fe13c8e4613519b4447123fc4546298"
+  end
+
+  resource "pygments" do
```

Comment:
Is this intentionally pinned on a fairly old version of `pygments`?

Reply:
Nope – I've updated the pinned versions.